statsforecast/core.py

```diff
@@ -198,51 +290,48 @@ def _get_n_jobs(n_groups, n_jobs, ray_address):
             actual_n_jobs = n_jobs
     return min(n_groups, actual_n_jobs)
 
-# Cell
+# %% ../nbs/core.ipynb 25
 class StatsForecast:
-
+    
     def __init__(
-            self,
-            df: pd.DataFrame, # DataFrame with columns `ds` and `y`, indexed by `unique_id`
-            models: List[Tuple[Callable, Any]], # List of tuples, each containing a fn and its parameters
+            self, 
```

Comment:
Why is df Optional?
I think that df should be the first entry for this class.

Unfortunately a lot of users are used to send the argument without naming first.